--- linux-falconwing_2.6.28.1.0.3454.bb	2011-05-25 13:38:04.000000000 +1000
+++ linux-falconwing_2.6.28.1.0.3454.bb.new	2011-05-25 13:40:46.000000000 +1000
@@ -15,5 +15,8 @@
 DEFAULT_PREFERENCE = "-1"
 DEFAULT_PREFERENCE_chumby-falconwing = "1"
 
-SRC_URI[md5sum] = "7ff9f40b4c08e7393098fe231197e0f0"
-SRC_URI[sha256sum] = "b6df540b1e450b791c659481d54e4187d554aef5fff12a004dda76ce92bd6413"
+# NOTE - these checksums were modified from the original distribution.
+# See:    https://github.com/clearwater/chumby-oe
+
+SRC_URI[md5sum] = "e4769e2f866d1f105cd7ad8f9f196aad"
+SRC_URI[sha256sum] = "11ad31a3d2487b48d9ff95df7494903d0fe109e9880a9569146f236703100a3a"
